test-sbt/jvm/src/main/scala/zio/test/runner/ZTestRunner.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test.runner
+
+import com.github.ghik.silencer.silent
+import sbt.testing._
+import zio.internal.{ Platform, PlatformLive }
+import zio.{ RIO, Runtime, ZIO }
+import zio.test._
+import zio.test.runner.ExecutedSpecStructure.Stats
+
+final class ZTestRunner(val args: Array[String], val remoteArgs: Array[String], testClassLoader: ClassLoader)
+    extends Runner {
+  def done(): String                           = "Done"
+  def tasks(defs: Array[TaskDef]): Array[Task] = defs.map(new ZTestTask(_, testClassLoader))
+}
+
+object ZTestRunner {
+  def apply(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): ZTestRunner =
+    new ZTestRunner(args, remoteArgs, testClassLoader)
+}
+
+class ZTestTask(val taskDef: TaskDef, testClassLoader: ClassLoader) extends Task {
+
+  private val platform: Platform = PlatformLive.makeDefault().withReportFailure(_ => ())
+
+  override def tags(): Array[String] = Array.empty
+
+  private def loadSpec[R, L]: RunnableSpec[R, L] = {
+    import scala.reflect.runtime.universe
```

Comment:
`scala-reflect` is not available on .js; might wanna look into https://github.com/portable-scala/portable-scala-reflect / look what the other test frameworks do right now.

Reply:
Looked at how `utest` did it and switched to `portable-scala-reflect`.
This required adding `@EnableReflectiveInstantiation` to `RunnableSpec`.